darknet_ros/include/darknet_ros/YoloObjectDetector.h

```diff
@@ -167,7 +167,6 @@ class YoloObjectDetector
   std::vector< std::vector<RosBox_> > rosBoxes_;
   std::vector<int> rosBoxCounter_;
   std::vector<cv::Scalar> rosBoxColors_;
-  darknet_ros_msgs::BoundingBoxes boundingBoxesResults_;
```

Comment:
The code can not compile because the compiler can not find the global variable boundingBoxesResults_. You deleted the variable in this line. Please readd the line again.